lib/pages/compare_pipeline_page.rb

```diff
@@ -38,9 +38,10 @@ def selected_pipeline_label_is?(label)
      end
 
      def get_pipeline_labels
-      pipeline_labels=[]
-      page.all('li.pim_list .pipeline_label'). find {|element| pipeline_labels.push(element.text.to_i)}
-      return pipeline_labels
+      @pipeline_labels=[]
```

Comment:
No need to define this as instance variable, since its scope is just with in the method. @ can be removed